Dockerfile

```
@@ -15,9 +19,18 @@ SHELL ["/bin/bash", "-c"]
 
 USER root
 # Update system for security checks
-RUN apt-get -y update && apt-get -y upgrade
+RUN apt-get -y update && apt-get -y upgrade vim jq curl
+
+SHELL ["/bin/bash", "-c"]
+RUN groupadd -g 1000 ${XCUBE_USER_NAME}
+RUN useradd -u 1000 -g 1000 -ms /bin/bash ${XCUBE_USER_NAME}
+RUN mkdir /workspace && chown ${XCUBE_USER_NAME}.${XCUBE_USER_NAME} /workspace
+RUN chown -R ${XCUBE_USER_NAME}.${XCUBE_USER_NAME} /opt/conda
+
+USER ${XCUBE_USER_NAME}
 
-USER xcube
+RUN source activate base && conda update -n base conda && conda init
+RUN source activate base && conda install -n base -c conda-forge mamba pip
```

```suggestion
RUN source activate base && conda install -n base -c conda-forge mamba=0.7.0 pip=21.3.1
```

Not vital, but makes conda faster and the build more reproducible.

---

I agree.